src/js/post-911-gib-status/components/EnrollmentPeriod.jsx

```diff
@@ -79,7 +76,7 @@ class EnrollmentPeriod extends React.Component {
     return (
       <div>
         <hr/>
-        <h4>{formatDateShort(enrollment.beginDate)} to {formatDateShort(enrollment.endDate)} at <span className="facility">{enrollment.facilityName.toLowerCase()}</span> ({enrollment.facilityCode})</h4>
+        <h4>{formatDateShort(enrollment.beginDate)} to {formatDateShort(enrollment.endDate)} at <span className="facility">{(enrollment.facilityName || '').toLowerCase()}</span> ({enrollment.facilityCode})</h4>
```

Comment:
Yeah, I was thinking the same thing. I'll open a ticket about that and try to enumerate the cases when an empty value would require a graceful fallback. Will ping it to you and then we can ask for design feedback if necessary.